packages/cmf-webpack-plugin/src/index.js

```diff
@@ -32,95 +12,85 @@ function ReactCMFWebpackPlugin(options = {}) {
 	this.lastRun = null;
 	this.lastWatch = null;
 	this.modifiedFiles = [];
-	this.options = Object.assign({
-		quiet: false,
-		watch: false,
-	}, options);
+	this.options = Object.assign(
+		{
+			quiet: false,
+			watch: false,
+		},
+		options,
+	);
 	this.log = (...args) => {
 		if (!this.options.quiet) {
 			console.error('[ReactCMFWebpackPlugin]', ...args); // eslint-disable-line no-console
 		}
 	};
 }
 
-ReactCMFWebpackPlugin.prototype.apply = function (compiler) {
+ReactCMFWebpackPlugin.prototype.apply = function reactCMFWebpackPluginApply(compiler) {
 	this.log('apply');
 
-	compiler.plugin('emit', (compilation, callback) => {
-		this.log('emit', JSON.stringify({ canRun: this.canRun, lastRun: this.lastRun, lastWatch: this.lastWatch }));
+	// adapt cmf settings result to output to /settings.json by default
+	let outputPath = compiler.options.output.path;
+	if (compiler.options.devServer && compiler.options.devServer.outputPath) {
+		outputPath = compiler.options.devServer.outputPath;
+	}
```

Comment:
IMHO if we required lodash we can do something a little more sexy, what do you think ?
const outpath = get(compiler.options, 'devServer.outputPath', compiler.options.output.path);